task/buildpacks/0.3/buildpacks.yaml

```diff
@@ -158,6 +163,10 @@ spec:
           mountPath: /layers
 
   volumes:
+    - name: buildpacks-trusted-ca
+      secret:
+        secretName: buildpacks-trusted-ca
+        optional: true
```

Comment:
sorry @MrMYHuang for looking at this late :sweat_smile: 
instead of mounting the secrets via volumes should we pass these via workspaces so that users are not bound to use particular secret name and will be optional as well? Moreover this will require a version bump as this is a feature addition. WDYT?

Reply:
Thank you. I will study it.

Reply:
Hello @vinamra28, I finish the rewrite. Please check commit https://github.com/tektoncd/catalog/pull/917/commits/65c3f503d4a9646aaaecc51ef35aa33f289734ea. Thank you.